extensions/azurecore/src/account-provider/azureAccountProviderService.ts

```diff
@@ -13,7 +11,8 @@ import * as path from 'path';
 import * as vscode from 'vscode';
 import CredentialServiceTokenCache from './tokenCache';
 import providerSettings from './providerSettings';
-import { AzureAccountProvider } from './azureAccountProvider';
+import { AzureAccountProvider as OldAzureAccountProvider } from './azureAccountProvider';
+import { AzureAccountProvider as NewAzureAccountProvider } from './azureAccountProvider2';
```

Comment:
I'd probably call `NewAzureAccountProvider ` something like `AzureAccountProvider`, and `OldAzureAccountProvider` something like `AzureAccountProviderDeprecated`.  That leaves a little less code to rewrite in the future.

Reply:
I'll rename these

Reply:
@aaomidi we have a feature flag called `workbench.enablePreviewFeatures`which will be useful here